src/components/Team/MemberCard.css

```diff
@@ -0,0 +1,76 @@
+.card {
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+    width: 425px;
+    height: 700px;
+    text-align: center;
+    flex: 0 1 24%;
+    padding: 50px;
+    box-sizing: border-box;;
+    align-content: center;
+    margin-bottom: 50px;
+  }
+
+.card:hover {
+    transform: translateY(-8px);
+    --box-shadow-color: #d15c19;
+    box-shadow: 0 10px 10px -10px var(--box-shadow-color);
+
+}
+
+h2 {
```

Comment:
It's generally preferable to avoid styling HTML tags directly because it can be difficult to tell where styles are coming from.